src/parseutils.jl

```diff
@@ -21,3 +21,5 @@ group(i, re::Regex, bytes, default) =
     re.ovec[2i+1] == Base.PCRE.UNSET ?
     default :
     SubString(bytes, re.ovec[2i+1]+1, prevind(bytes, re.ovec[2i+2]+1))
+
+lceq(a, b) = lowercase(a) == lowercase(b)
```

Comment:
This was previously a non-exported function in Messages.jl, and was only ever used a few lines away from its definition. If we're going to have it be an exported part of the Parsers.jl interface then it should have a less abbreviated name and docstrings and tests.

The name should probably reflect the fact that it is only intended for comparing HTTP header `field-name`s, and the doc string should mention that `field-name` is ASCII-only.

`headerequal`?

Reply:
Will be addressed by #310